### PR TITLE
Error message

### PR DESCRIPTION
**Pull request by TrevorOctober**
@hodgestar Please review cc @KaitCrawford

### CODE REVIEW

**Comment by hodgestar** [comment]
We should add tests for the error cases in this PR too.

**Comment by TrevorOctober** [comment]
@hodgestar Ready for review

File: mentorship/mentor/tests/test_views.py [hunk 1]
```diff
@@ -300,6 +300,13 @@ def mk_tiniest_png(self, name):
         data = base64.b64decode(png64)
         return SimpleUploadedFile(name, data, content_type='image/png')
 
+    def mk_tiniest_jpg(self, name):
+        png64 = (
+            "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAA"
+            "CklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==")
```
Comment (hodgestar) [inline mentorship/mentor/tests/test_views.py]: This data does not describe a JPEG. I would rename the function `mk_invalid_jpg` and just return:
```python
return SimpleUploadedFile(name, data='invalid data', content_type='image/jpeg')
```
Note that the correct MIME type for JPEGs is `image/jpeg`.

**Comment by TrevorOctober** [comment]
@hodgestar Ready for re-review

File: mentorship/mentor/tests/test_views.py [hunk 2]
```diff
@@ -300,6 +300,13 @@ def mk_tiniest_png(self, name):
         data = base64.b64decode(png64)
         return SimpleUploadedFile(name, data, content_type='image/png')
 
+    def mk_invalid_jpg(self, name):
+        png64 = (
+            "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAA"
+            "CklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==")
+        data = base64.b64decode(png64)
```
Comment (hodgestar) [inline mentorship/mentor/tests/test_views.py]: You can just pass `data='invalid data'` into `SimpleUploadFile`. You don't need to use the PNG data.

**Comment by TrevorOctober** [comment]
@hodgestar Please review

File: mentorship/mentor/views.py [hunk 3]
```diff
@@ -415,13 +415,13 @@ class ProfileViewSet(mixins.ListModelMixin, mixins.RetrieveModelMixin,
...
         if 'image' not in self.request.data:
-            # TODO: add error message
+            raise ValidationError("Please choose an image")
             return response.Response(
                 status=status.HTTP_400_BAD_REQUEST,
                 content="XXX")
         profile_pic = request.data['image']
         if profile_pic.content_type != 'image/png':
-            # TODO: add error message
+            raise ValidationError("Image must be png")
```
Comment (hodgestar) [inline mentorship/mentor/views.py]: The error message should explain a little more clearly what the error is. E.g. `The uploaded image must have content type 'image/png'.`.

File: mentorship/mentor/views.py [hunk 4]
```diff
@@ -415,13 +415,13 @@ class ProfileViewSet(mixins.ListModelMixin, mixins.RetrieveModelMixin,
     @parser_classes((FormParser, MultiPartParser))
     def image(self, request, *args, **kw):
         if 'image' not in self.request.data:
-            # TODO: add error message
+            raise ValidationError("Please choose an image")
```
Comment (hodgestar) [inline mentorship/mentor/views.py]: The error message should explain a little more clearly what the error is. E.g. `Please upload a PNG under a variable named 'image' encoded using either the application/x-www-form-urlencoded or multipart/form-data content types.`.

**Comment by hodgestar** [comment]
:+1: once Travis builds successfully.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**